extensions/dacpac/src/test/index.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const path = require('path');
+import * as testRunner from 'vscodetestcover';
+
+const suite = 'DacFx Tests';
+
+const testOptions: any = {
+	ui: 'bdd',
+	useColors: true,
+	timeout: 600000
```

Comment:
do you really need the timeout to be so long? 60 seconds should be more than enough.